source/index.html.md.erb

```diff
@@ -1,21 +1,33 @@
 ---
-title: GDS technical guidance
+title: The GDS Way
 ---
 
 # <%= current_page.data.title %>
 
+# What is The GDS Way?
+
+Since GDS was formed a few years ago, we've grown to an organisation of several hundred people with well established teams which are meeting user needs by creating software.
+
+As we've grown the teams have all had to choose how they want to build their services. In some ways this has been good - every team has picked the most suitable tools for their project - but now we suspect that we're at the point where having some easy-to-use defaults for new teams will be helpful. Some of the things we're hoping to achieve are:
+
+* using similar tools across the organisation will make people more productive when they move between teams, meaning we might be able to do that faster and more often (assuming people want to)
```

Comment:
Thanks, I wanted to keep the content slightly informal in keeping with GDS culture.